kikuchipy/signals/tests/test_ebsd_masterpattern.py

```diff
@@ -35,7 +39,14 @@
 from kikuchipy.signals.ebsd_master_pattern import (
     EBSDMasterPattern,
     LazyEBSDMasterPattern,
+    _get_direction_cosines,
+    _get_lambert_interpolation_parameters,
+    _get_patterns_chunk,
+    _min_number_of_chunks,
+    HemisphereError,
 )
+from kikuchipy.signals.ebsd import LazyEBSD, EBSD
+from kikuchipy.indexing.similarity_metrics import _SIMILARITY_METRICS
```

Comment:
Now you can import `from kikuchipy.indexing.similarity_metrics import ncc, ndp` directly instead.